test/cmd/admin.sh

```diff
@@ -41,188 +42,192 @@ status:
   - lastHeartbeatTime: 2015-09-08T16:58:02Z
     lastTransitionTime: 2015-09-04T11:49:06Z
     reason: kubelet is posting ready status
-    status: "True"
+    status: \"True\"
     type: Ready
-' | oc create -f -
-
-[ "$(oadm manage-node --selector='' --schedulable=true | grep --text 'Ready' | grep -v 'Sched')" ]
```

Comment:
I don't really understand why `--text` was here, so I removed it. This is not going through any binaries, so this is arguably more correct?